src/imageio.cs

```diff
@@ -2870,6 +2874,26 @@ interface CGImageThumbnailOptions {
 		IntPtr kCGImageSourceSubsampleFactor { get; }
 	}
 
+	[Partial]
+	[Watch (10, 0), TV (17, 0), Mac (14, 0), iOS (17, 0), MacCatalyst (17, 0)]
+	interface CGImageDecodeOptions {
+		[Internal]
+		[Field ("kCGImageSourceDecodeRequest")]
+		IntPtr kDecodeRequest { get; }
```

Comment:
```suggestion
		IntPtr DecodeRequest { get; }
```

Reply:
@rolfbjarne since these fields are internal, wouldn't we want to keep the name as is?

Reply:
As a general rule, I think it's best to always use the (potential) public name, just in case someone wants to make it public (in which case they'd have to remember to fix the name, which is easily forgotten).